mage_ai/command_line.py

```diff
@@ -40,6 +40,20 @@ def main():
         pipeline = Pipeline(pipeline_uuid, project_path)
 
         asyncio.run(pipeline.execute(analyze_outputs=False, update_status=False))
+    elif command == 'test':
```

Comment:
I moved the import statement under each command statement to reduce dependency loading time. So you might want to import Pipeline in this branch.